frontend/src/components/Register.vue

```diff
@@ -13,16 +13,16 @@
                    required autofocus v-model="addUserForm.email">
           </div>
           <div class="form-label-group">
-            <label for="inputUsername">Username</label>
-            <input type="username" id=" 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" class="form-control"
+            <label for="inputPassword">Username</label>
```

Comment:
Això no hauria de ser "inputUsername"?
```suggestion
            <label for="inputUsername">Username</label>
```

Reply:
@aliciacarrascoo @PauRodriguezTrens @mariaroman99